base_layer/core/src/chain_storage/horizon_data.rs

```diff
@@ -45,3 +45,17 @@ impl HorizonData {
         &self.utxo_sum
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    #[test]
+    fn coverage_horizon_data() {
```

Comment:
See comment above. 
Here should be easily able to confirm that we do indeed return 0.h and 0.g